package.json

```diff
@@ -14,6 +14,16 @@
     "import": "./dist/index.js",
     "require": "./dist/index.cjs"
   },
+  "tags": [
+    "url",
+    "urlpattern",
+    "url-pattern",
+    "type-script",
```

Comment:
why not typescript as one word?

Reply:
I think its better to drop it completely in hind-sight.